docs/_sass/colors.scss

```diff
@@ -0,0 +1,139 @@
+// This is forked. Do not edit.
```

Comment:
why not? It it's not editable it shouldn't be in the repo.

Reply:
I'm not sure what your objection is... it's pretty standard practice to put warnings on files that are derived from other sources. Technically such files can be edited, but it's not recommended because it's much harder to maintain.